api-testing/multicloud-api/build.gradle

```diff
@@ -0,0 +1,24 @@
+plugins {
+    id 'java'
+}
+
+group 'sodafoundation.io'
+version '1.0.0'
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    testImplementation 'org.junit.jupiter:junit-jupiter-api:5.6.0'
+    testRuntimeOnly 'org.junit.jupiter:junit-jupiter-engine'
+    testImplementation('junit:junit:4.13')
```

Comment:
How are we decide the type of dependencies?

Reply:
Type means what?

Reply:
Type mean how are you deciding whether these dependencies are compile, test, implementation type.
You can refer the following link.
[https://docs.gradle.org/current/userguide/declaring_dependencies.html](url)

Reply:
Already mention in dependency  "testImplementation", ` testImplementation 'org.junit.jupiter:junit-jupiter-api:5.6.0'`